new.go

```diff
@@ -282,13 +283,19 @@ func newBuilder(ctx context.Context, store storage.Store, options BuilderOptions
 	coptions.IDMappingOptions = newContainerIDMappingOptions(options.IDMappingOptions)
 
 	container, err := store.CreateContainer("", []string{name}, imageID, "", "", &coptions)
-	suffix := 1
+	suffix := rand.Int()
+	tries := 0
 	for err != nil && errors.Cause(err) == storage.ErrDuplicateName && options.Container == "" {
-		suffix++
+		tries++
 		tmpName := fmt.Sprintf("%s-%d", name, suffix)
 		if container, err = store.CreateContainer("", []string{tmpName}, imageID, "", "", &coptions); err == nil {
 			name = tmpName
 		}
+		if tries == 100 {
+			logrus.Debug("reached maximum of 100 ties to create working container")
```

Comment:
This should probably be a return err, so that the  user knows it was attempted 100 times.

Reply:
I'll fix that and will also change the 100 *ties* to 100 attempts :laughing: